ci/requirements-optional-conda.txt

```diff
@@ -1,28 +1,27 @@
-pytest>=3.1.0
-pytest-cov
-pytest-xdist
-flake8
-sphinx=1.5*
-nbsphinx
-ipython
-python-dateutil
-pytz
-openpyxl
-xlsxwriter
-xlrd
-xlwt
-html5lib
-patsy
 beautifulsoup4
-numpy
-cython
-scipy
+blosc
+bottleneck
+fastparquet
+feather-format
+html5lib
+ipython
+ipykernel
+jinja2
+lxml
+matplotlib
+nbsphinx
```

Comment:
Heads up: `moto` is missing from this list and isn't a secondary dependency of any other package:

```bash
conda create -y -n pandas_dev python=3 -c conda-forge --file ci/requirements_all.txt
conda install -y -n pandas_dev -c conda-forge fastparquet feather-format ipykernel pandas-datareader pandas-gbq python-blosc pyarrow psycopg2 pymysql s3fs xarray
source activate pandas_dev
python setup.py develop
pip freeze --local
```

<details>

alabaster==0.7.10
apipkg==1.4
appnope==0.1.0
asn1crypto==0.22.0
Babel==2.5.1
backports.functools-lru-cache==1.4
beautifulsoup4==4.6.0
bleach==2.0.0
blosc==1.4.4
bokeh==0.12.10
boto3==1.4.7
botocore==1.5.92
Bottleneck==1.2.1
certifi==2017.7.27.1
cffi==1.10.0
chardet==3.0.4
click==6.7
cloudpickle==0.4.0
coverage==4.4.1
cryptography==2.0.3
cycler==0.10.0
Cython==0.27.2
dask==0.15.4
decorator==4.1.2
distributed==1.19.3
docutils==0.14
entrypoints==0.2.3
et-xmlfile==1.0.1
execnet==1.5.0
fastparquet==0.1.3
feather-format==0.4.0
flake8==3.5.0
google-api-python-client==1.6.4
h5netcdf==0.5.0
h5py==2.7.1
heapdict==1.0.0
html5lib==0.999999999
httplib2==0.10.3
idna==2.6
imagesize==0.7.1
ipykernel==4.6.1
ipython==6.2.1
ipython-genutils==0.2.0
jdcal==1.3
jedi==0.10.2
Jinja2==2.9.6
jmespath==0.9.3
jsonschema==2.6.0
jupyter-client==5.1.0
jupyter-core==4.3.0
llvmlite==0.20.0
locket==0.2.0
lxml==4.1.0
MarkupSafe==1.0
matplotlib==2.1.0
mccabe==0.6.1
mistune==0.8
msgpack-python==0.4.8
nbconvert==5.3.1
nbformat==4.4.0
nbsphinx==0.2.14
netCDF4==1.3.0
numba==0.35.0+6.gaa35fb1
numexpr==2.6.4
numpy==1.13.3
oauth2client==4.1.2
openpyxl==2.5.0b1
pandas==0.21.0
pandas-datareader==0.5.0
pandas-gbq==0.2.0
pandocfilters==1.4.1
partd==0.3.8
patsy==0.4.1
pexpect==4.2.1
pickleshare==0.7.4
prompt-toolkit==1.0.15
psutil==5.4.0
psycopg2==2.7.3.1
ptyprocess==0.5.2
py==1.4.34
pyarrow==0.7.1
pyasn1==0.3.7
pyasn1-modules==0.1.4
pycodestyle==2.3.1
pycparser==2.18
pyflakes==1.6.0
Pygments==2.2.0
PyMySQL==0.7.11
pyOpenSSL==17.2.0
pyparsing==2.2.0
PySocks==1.6.7
pytest==3.2.3
pytest-cov==2.5.1
pytest-forked==0.2
pytest-xdist==1.20.1
python-dateutil==2.6.1
pytz==2017.3
PyYAML==3.12
pyzmq==16.0.2
requests==2.18.4
requests-file==1.4.1
requests-ftp==0.3.1
rsa==3.4.2
s3fs==0.1.2
s3transfer==0.1.11
scipy==0.19.1
seaborn==0.8.1
simplegeneric==0.8.1
simplejson==3.11.1
six==1.11.0
snowballstemmer==1.2.1
sortedcontainers==1.5.7
Sphinx==1.5.6
SQLAlchemy==1.1.13
statsmodels==0.8.0
tables==3.4.2
tblib==1.3.2
testpath==0.3.1
thrift==0.10.0
toolz==0.8.2
tornado==4.5.2
traitlets==4.3.2
uritemplate==3.0.0
urllib3==1.22
wcwidth==0.1.7
webencodings==0.5
xarray==0.9.6
xlrd==1.1.0
XlsxWriter==1.0.2
xlwt==1.3.0
zict==0.1.3

</details>